### PR TITLE
{Doc} Update doc/try_new_features_before_release.md

### PR DESCRIPTION
**Pull request by jiasli**
**Description**  
Update `doc/try_new_features_before_release.md`.

### CODE REVIEW

**Comment by jiasli** [comment]
Adding me @qwordy, @dbradish-microsoft as the code owners for `/doc/`. cc @yungezz @yonzhan

File: doc/try_new_features_before_release.md [hunk 1]
```diff
@@ -57,19 +58,26 @@ env\Scripts\activate.ps1
...
+# Create a Python virtual environment
+# https://docs.python.org/3/using/cmdline.html#cmdoption-m
 python -m venv env
-# Activate it
+
+# Activate the virtual environment
 env\Scripts\activate.ps1
-# Install azdev
+
+# Install azdev - the dev tool for Azure CLI
```
Comment (qwordy) [inline doc/try_new_features_before_release.md:75]:
```suggestion
# Install azdev - the development tool for Azure CLI
```

**Review by yonzhan — APPROVED** [review]
LGTM

**Comment by yonzhan** [comment]
Doc

**Review by qwordy — APPROVED** [review]

**Pull request merged**